include/glow/Backends/Backend.h

```diff
@@ -71,9 +71,12 @@ class Backend {
   }
   /// @}
 
-  /// \returns true if backend supports given kind of operation with
-  /// the given \p elementTy element type.
-  virtual bool isOpSupported(Kinded::Kind opKind, ElemKind elementTy) const = 0;
+  /// \returns whether the provided \p NI is supported by the backend.
+  virtual bool isOpSupported(Function *F, const NodeInfo &NI) const = 0;
+
+  /// \returns whether the provided \p NI is supported by the backend via
+  /// lowering the \p NI into other NodeInfos, matching the lowering logic.
```

Comment:
It would worthwhile to mention here that if node cannot be lowered than it's not `isOpSupportedViaLowering` even though the node itself could be supported otherwise.